packages/shared/typedoc.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://typedoc.org/schema.json",
-  "entryPoints": ["./src/index.ts", "./src/react/index.ts", "./src/react/types.ts"],
+  "entryPoints": ["./src/index.ts", "./src/react/types.ts", "./src/react/hooks/*.{ts,tsx}"],
```

Comment:
I needed access to the exported interfaces inside `useOrganization`/`useOrganizationList` but didn't want to change the public API. So we're looking at the hooks individually now.